setup.py

```diff
@@ -19,7 +19,7 @@
     install_requires=[
         "toolz>=0.9.0,<1.0.0;implementation_name=='pypy'",
         "cytoolz>=0.9.0,<1.0.0;implementation_name=='cpython'",
-        "eth-abi>=1.1.1,<2",
+        "eth-abi==2.0.0a1",
```

Comment:
We shouldn't require people to upgrade. We especially shouldn't require people to install an alpha dependency in a stable release. `eth-abi>=1.1.1,<3` seems like a reasonable choice